charts/nri-bundle/values.yaml

```diff
@@ -1,106 +1,129 @@
+  licenseKey: ""
+  # -- The license key for your New Relic Account. This will be preferred configuration option if both `insightsKey` and `customSecret` are specified.
+  insightsKey: ""
+  # -- Name of the Secret object where the license key is stored
+  customSecretName: ""
+  # -- Key in the Secret object where the license key is stored
+  customSecretLicenseKey: ""
+  # -- Key in the Secret object where the insights key is stored
+  customSecretInsightsKey: ""
+  # -- Additional labels for chart pods
+
+  podLabels: {}
+  # -- Additional labels for chart objects
+  labels: {}
+
+  images:
+    # -- Changes the registry where to get the images. Useful when there is an internal image cache/proxy
+    registry: ""
+    # -- Set secrets to be able to fetch images
+    pullSecrets: []
+
+  serviceAccount:
+    # -- Add these annotations to the service account we create
+    annotations: {}
+    # -- Configures if the service account should be created or not
+    create:
+    # -- Change the name of the service account. This is honored if you disable on this cahrt the creation of the service account so you can use your own
+    name:
+
+  # -- (bool) Sets pod's hostNetwork
+  # @default -- false
+  hostNetwork:
+  # -- Sets pod's dnsConfig
+  dnsConfig: {}
+
+  # -- Sets pod's priorityClassName
+  priorityClassName: ""
+  # -- Sets security context (at pod level)
+  podSecurityContext: {}
+  # -- Sets security context (at container level)
+  containerSecurityContext: {}
+
+  # -- Sets pod/node affinities
+  affinity: {}
+  # -- Sets pod's node selector
+  nodeSelector: {}
+  # -- Sets pod's tolerations to node taints
+  tolerations: []
+
+  # -- Adds extra attributes to the cluster and all the metrics emitted to the backend
+  customAttributes: {}
+
+  # -- (bool) Reduces number of metrics sent in order to reduce costs
+  # @default -- false
+  lowDataMode:
+
+  # -- (bool) In each integration it has different behavior. See [Further information](#values-managed-globally-3) but all aims to send less metrics to the backend to try to save costs |
+  # @default -- false
+  privileged:
+
+  # -- (bool) Must be set to `true` when deploying in an EKS Fargate environment
+  # @default -- false
+  fargate:
+
+  # -- Configures the integration to send all HTTP/HTTPS request through the proxy in that URL. The URL should have a standard format like `https://user:password@hostname:port`
+  proxy: ""
+
+  # -- (bool) Send the metrics to the staging backend. Requires a valid staging license key
+  # @default -- false
+  nrStaging:
+  fedramp:
+    # fedramp.enabled -- (bool) Enables FedRAMP
+    # @default -- false
+    enabled:
+
+  # -- (bool) Sets the debug logs to this integration or all integrations if it is set globally
+  # @default -- false
+  verboseLog:
+
+# To add values to the subcharts. Follow Helm's guide: https://helm.sh/docs/chart_template_guide/subcharts_and_globals
```

Comment:
even if we do not have to, since we are good people, I would add an example to instrument the redis integration. (the old one maybe should be updated with the new structure, we can copy it from the canaries)

Since that is always difficult and we would avoid a lot of issues/questions